src/OpenIddict.Client.WebIntegration/OpenIddictClientWebIntegrationProviders.xml

```diff
@@ -1207,6 +1207,24 @@
     </Setting>
   </Provider>
 
+  <!--
+                                           ▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄
+                                           ██ ███ █ ▄▄▄██ ▄▄▀█▄ ▄██ ▄▀▄ █▄ ▄██
+                                           ███ █ ██ ▄▄▄██ ▀▀▄██ ███ █ █ ██ ███
+                                           ███▄▀▄██ ▀▀▀██ ██ █▀ ▀██ ███ █▀ ▀██
+                                           ▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀
+  -->
+
+  <Provider Name="Verimi" Documentation="https://docs.verimi.de/">
```

Comment:
Starting with OpenIddict 4.5, each provider must be assigned an `Id` property (since the provider name can now be customized by the developer and multiple instances of the same provider type can be added).

E.g: `de781ebe-164c-4948-96d8-5e5adbbf19f0`.

Reply:
For the documentation URL, `https://docs.verimi.de/#/oidc/oidc_overview` sounds like a better candidate as it directly gives you access to all the OIDC details, including the address of the production and staging environments.

Reply:
Thanks for your feedback. I added the required ID and updated the documentation URL.